src/Microsoft.Diagnostics.Monitoring.RestServer/MetricsOptions.cs

```diff
@@ -51,5 +52,13 @@ public MetricsOptions()
         public int UpdateIntervalSeconds { get; set; }
 
         public int MetricCount { get; set; }
+
+        public List<MetricProvider> Providers { get; set; } = new List<MetricProvider>(0);
+    }
+
+    public class MetricProvider
+    {
+        public string Provider { get; set; }
```

Comment:
nit: ProviderName; makes it clear that it is the name of the provider and not some object with additional properties (also reserves Provider to be used as an object in the future if necessary).